charts/latest/azurefile-csi-driver/templates/csi-azurefile-driver.yaml

```diff
@@ -15,3 +15,5 @@ spec:
     - Persistent
     - Ephemeral
   fsGroupPolicy: {{ .Values.feature.fsGroupPolicy }}
+  tokenRequests:
```

Comment:
this value is not used

Reply:
This field is needed when using csi driver's token request feature: https://kubernetes-csi.github.io/docs/token-requests.html#example